src/library/dlgtrackinfo.cpp

```diff
@@ -504,9 +505,9 @@ void DlgTrackInfo::focusField(const QString& property) {
     if (property.isEmpty()) {
         return;
     }
-    QWidget* w = m_propertyWidgets.find(property).value();
-    if (w) {
-        w->setFocus();
+    auto it = m_propertyWidgets.constFind(property);
+    if (it == m_propertyWidgets.constEnd()) {
```

Comment:
major typo flaw. Did you forget to test this?
```suggestion
    if (it != m_propertyWidgets.constEnd()) {
```

Reply:
I reverted early return just before pushing without testing again..
fixed.